bean-utils-library/src/test/java/com/hotels/beans/BeanUtilsTest.java

```diff
@@ -109,6 +113,26 @@ public void testGetTransformerFunctionWorksProperly(final String testCaseDescrip
                 .forEach(i -> assertThat(actual.get(i), sameBeanAs(fromFooSimpleList.get(i))));
     }
 
+    /**
+     * Test that the transformer function is applied earlier than the default value
+     */
+    @Test
+    public void testTransformerFunctionHasHigherPriorityThanDefaultValue() {
+        //GIVEN
+        BeanUtils beanUtils = new BeanUtils();
+        FromFooSimpleBooleanField fromFooSimpleNullFields = createFromFooSimpleNullFields();
+        FieldTransformer<Boolean, Boolean> nullToTrue =
+                new FieldTransformer<>("work", aBoolean -> aBoolean == null || aBoolean);
+
+        //WHEN
+        ImmutableToFooSimpleBoolean transformed = beanUtils.getTransformer()
```

Comment:
to be coherent with the other test, could you kindly use `actual` instead of `transformed`?
thanks